clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -313,7 +313,7 @@ public void onFailure(RuntimeException e) {
                 metadata());
 
         // create the request for the coordinator
-        log.debug("Issuing request ({}: {}) to coordinator {}", ApiKeys.JOIN_GROUP, request, this.coordinator.id());
+        log.debug("Sending JoinGroup ({}) to coordinator {}", request, this.coordinator.id());
```

Comment:
The coordinator id ends up being a really big negative integer, but if we used host:port in the log message then maybe it wouldn't be too bad? We'd probably want to make the same change in the other messages as well.